Kernel/FileSystem/OpenFileDescription.h

```diff
@@ -40,8 +40,8 @@ class OpenFileDescription : public RefCounted<OpenFileDescription> {
 
     void set_rw_mode(int options)
     {
-        set_readable(options & O_RDONLY);
-        set_writable(options & O_WRONLY);
+        set_readable((options & O_RDONLY) != 0);
```

Comment:
Prefer the `== O_RDONLY` variant, as it does not require the flag being checked to be exactly one bit (or the code reader to know / think about that).

Reply:
Sounds reasonable, I've converted the conditions I changed (at least in the kernel) to that style.